draft-ietf-quic-recovery.md

```diff
@@ -1190,9 +1193,13 @@ SetLossDetectionTimer():
     return
 
   // Don't arm timer if there are no ack-eliciting packets
-  // in flight and the handshake is complete.
-  if (endpoint is client with 1-RTT keys &&
-      no ack-eliciting packets in flight):
+  // in flight and the endpoint is a server.  Do arm the
+  // timer until the client has received a Handshake ACK
+  // or has completed the handshake.
+  if (no ack-eliciting packets in flight &&
+      (endpoint is server ||
+       has 1-RTT keys ||
+       has received Handshake ACK)):
```

Comment:
The prose I would use here is:

An endpoint always keeps the timer running if it has an ack-eliciting packet that has not been acknowledged.

A client keeps the timer running during the handshake until it has confirmed that the server is no longer limited in how much it can send.  The server needs to acknowledge a Handshake packet or a 1-RTT packet from the client to demonstrate this.